conductor-client/src/main/java/com/openlattice/postgres/PostgresColumn.java

```diff
@@ -400,6 +400,8 @@
     public static final String                   PERMISSIONS_FIELD                 = "permissions";
     public static final PostgresColumnDefinition PERMISSIONS                       =
             new PostgresColumnDefinition( PERMISSIONS_FIELD, TEXT_ARRAY );
+    public static final String                   PG_SCHEMA_NAME                    = "schemaname";
+    public static final String                   PG_TABLE_NAME                     = "tablename";
```

Comment:
not at all important, but these feel like they should live somewhere separate since they're postgres internal values and are not being used to construct a PostgresColumnDefinition

Reply:
Oh yeah this may be better than my comment

Reply:
I agree, just didn't know where to put them. I can create a separate class/

Reply:
Yeah probably a PostgresInternalColumns.kt?